src/modules/Remote.rb

```diff
@@ -72,38 +74,46 @@ def main
 
     # Checks if remote administration is currently allowed
     def IsEnabled
-      @allow_administration
+      !IsDisabled()
     end
 
     # Checks if remote administration is currently disallowed
     def IsDisabled
-      !IsEnabled()
+      @mode == :disabled
     end
 
     # Enables remote administration.
-    def Enable
-      @allow_administration = true
+    def Enable(use_vncmanager = false)
```

Comment:
this boolean option is bad. It makes code hardly readable and nonintuitive as you can see few lines bellow (https://github.com/yast/yast-network/pull/401/files#diff-3e7dc301d1e2f6c63659b651f1c3ed2dR161)

I'd stay with ```Enable``` with no option for xinetd to stay backward compatible and introduce e.g. ```enable_with_persistent_sessions``` for vncmanager.